examples/module1/lesson1/validate-it/index.html

```diff
@@ -18,20 +18,20 @@
         <div class="grid grid-cols-2 gap-x-2">
           <button
             class="rounded-md text-lg bg-blue-200 hover:bg-blue-800 hover:text-white p-4 mt-4"
-            id="button"
+            id="validateButton"
           >
             Validate It!
           </button>
           <button
             class="rounded-md text-lg bg-red-200 hover:bg-red-800 hover:text-white p-4 mt-4"
-            id="button2"
+            id="resetButton"
```

Comment:
trzymał bym się konsekwentnie nazewnictwa
`validateButton` => `Validate It`

`clearButton` => `Clear!`   (zamiast `resetButton`)

ale to pierdoła